### PR TITLE
ORG-9 | Eslint, prettier and tsconfig file setup

### PR DESCRIPTION
**Pull request by TomaszewskiMichal**
https://travel-instagram.atlassian.net/browse/ORG-9

### CODE REVIEW

**Review by teo0098 — CHANGES_REQUESTED** [review]
I reckon that it would be a good idea if we split packages for those that are needed for production (dependencies) and for those that we will be utilizing in development (dev dependencies) Now we hold each package in dependencies and for instance @types/node will not be used in production so it should go to dev dependencies. 

The second remark involves tsconfig.json. You use extends property although there is duplication in terms of properties used in root tsconfig.json and in next.js tsconfig.json.

File: .prettierrc.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "printWidth": 120,
+  "singleQuote": true
```
Comment (Parodix007) [inline .prettierrc.json:4]: I would rather go with singleQuotes false and semicolin true

Comment (teo0098) [reply .prettierrc.json:4]: Airbnb one the style guides actually recommends using single quotes

Comment (TomaszewskiMichal) [reply .prettierrc.json:4]: main thing in this pr was to setup it, we need to add more rules, then we can talk about it, and agree which to choose

**Review by teo0098 — APPROVED** [review]

**Pull request merged**